jenkins/ocp-config/build/bc.yml

```diff
@@ -21,9 +21,25 @@ parameters:
 - name: JENKINS_MASTER_BASE_FROM_IMAGE
   required: true
   description: Base image for Jenkins master
+- name: JENKINS_MASTER_CPU_REQUEST
+  value: 200m
+- name: JENKINS_MASTER_CPU_LIMIT
+  value: "1"
+- name: JENKINS_MASTER_MEMORY_REQUEST
+  value: 1Gi
+- name: JENKINS_MASTER_MEMORY_LIMIT
+  value: 2Gi  
 - name: JENKINS_AGENT_BASE_FROM_IMAGE
   required: true
   description: Base image for Jenkins agent base
+- name: JENKINS_AGENT_BASE_CPU_REQUEST
+  value: 200m
+- name: JENKINS_AGENT_BASE_CPU_LIMIT
+  value: "1"
+- name: JENKINS_AGENT_BASE_MEMORY_REQUEST
+  value: 1Gi
+- name: JENKINS_AGENT_BASE_MEMORY_LIMIT
```

Comment:
Note that here we are setting the resources for the container image build, not for the deployed container. To avoid any confusion and/or duplication of the parameters in `ods-core.env`, I would welcome if we name these e.g. `JENKINS_MASTER_BUILD_CPU_REQUEST` etc.